### PR TITLE
Master hr holidays split rli

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web#id=46106&view_type=form&model=project.task&action=333&active_id=965&menu_id=4720

### CODE REVIEW

File: addons/hr_holidays/report/hr_leave_reports.xml [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0"?>
+<odoo>
+        <record id="view_holiday_pivot" model="ir.ui.view">
```
Comment (tde-banana-odoo) [inline addons/hr_holidays/report/hr_leave_reports.xml]: Side note: it seems there is an indentation issue, isn't it ?

File: addons/hr_holidays/report/hr_leave_reports.xml [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            <field name="name">hr.holidays.report_pivot</field>
+            <field name="model">hr.leave</field>
```
Comment (tde-banana-odoo) [inline addons/hr_holidays/report/hr_leave_reports.xml]: hr.leave views should stay in hr_leave_views file. Those are not really report view, only classic views of pivot or graph type.

File: addons/hr_holidays/views/hr_leave_views.xml [hunk 3]
```diff
@@ -0,0 +1,382 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<odoo>
+
+        <!-- Graph report -->
+        <record id="view_evaluation_report_graph" model="ir.ui.view">
```
Comment (tde-banana-odoo) [inline addons/hr_holidays/views/hr_leave_views.xml]: Side note: It seems threre is an indentation issue.

File: addons/project_timesheet_holidays/models/account_analytic.py [hunk 4]
```diff
@@ -8,7 +8,8 @@
 class AccountAnalyticLine(models.Model):
     _inherit = 'account.analytic.line'
 
-    holiday_id = fields.Many2one("hr.holidays", string='Leave Request')
+    # TODO: See if it is the right type
```
Comment (tde-banana-odoo) [inline addons/project_timesheet_holidays/models/account_analytic.py]: TODO: solve the todo :p

**Comment by tde-banana-odoo** [comment]
Hello,

About commits: I think you can merge split and renaming since there are already some new models in the split.

For other commits, do not forget to put the addon name ([REM] hr_holidays: remove unused report"). Don't forget to explain a bit why you did the changes, not only what or how. For example for the split, the purpose is to allow different business code for separate models as allocation and requests are not always the same; that it will add some code but simplify future improvements; ...

Thanks !

**Comment by smetl** [comment]
Is there a reason this PR is merged but still "Open"?

**Comment by tde-banana-odoo** [comment]
We are waiting for Cthulhu to wake up.

**Pull request closed without merging**